apps/condo/domains/analytics/tasks/helpers/sendDailyStatistics.spec.js

```diff
@@ -0,0 +1,637 @@
+/**
+ * @jest-environment node
+ */
+const index = require('@app/condo/index')
+const { faker } = require('@faker-js/faker')
+const dayjs = require('dayjs')
+
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { setFakeClientMode, setFeatureFlag, makeLoggedInAdminClient, waitFor } = require('@open-condo/keystone/test.utils')
+const { i18n } = require('@open-condo/locales/loader')
+
+const { SEND_DAILY_STATISTICS_TASK, SEND_DAILY_STATISTICS_ORGANIZATIONS_ENABLED } = require('@condo/domains/common/constants/featureflags')
+const { SEND_DAILY_STATISTICS_MESSAGE_TYPE } = require('@condo/domains/notification/constants/constants')
+const { Message } = require('@condo/domains/notification/utils/testSchema')
+const { makeEmployeeUserClientWithAbilities, createTestOrganizationEmployee, createTestOrganizationEmployeeRole } = require('@condo/domains/organization/utils/testSchema')
+const { makeClientWithProperty } = require('@condo/domains/property/utils/testSchema')
+const { INCIDENT_STATUS_NOT_ACTUAL } = require('@condo/domains/ticket/constants/incident')
+const { STATUS_IDS } = require('@condo/domains/ticket/constants/statusTransitions')
+const { updateTestTicket, createTestIncidentProperty } = require('@condo/domains/ticket/utils/testSchema')
+const {
+    createTestTicket, TicketStatus, createTestIncident, createTestIncidentClassifierIncident, IncidentClassifier,
+} = require('@condo/domains/ticket/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+
+const { UserDailyStatistics, sendDailyMessageToUserSafely } = require('./sendDailyStatistics')
+
+
+describe('sendDailyStatistics', () => {
+    let admin, statuses,
+        allowedIncidentClassifiers, notAllowedIncidentClassifiers
+
+    setFakeClientMode(index)
+    jest.setTimeout(10000)
```

Comment:
In CI tests are run with timeouts of 15 seconds (`testTimeout=15000` from `run_condo_domain_tests.sh`)

So if I'm not mistaken, `jest.setupSpecs.js` will only be relevant for local execution

Reply:
Why do you need to rest timeout here?

Reply:
> Why do you need to rest timeout here?

Because I was locally missing 5 seconds, which are set in `jest.setupSpecs.js`. 
But in CI we have 15 sec, so it's not a problem